### PR TITLE
en: update doc to enable cross-zone load balancing

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update doc to enable cross-zone load balancing for NLB.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/277
- Other reference link(s):

### CODE REVIEW

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
     kubectl --kubeconfig credentials/kubeconfig_${eks_name} create -f db-monitor.yaml -n ${namespace}
     ```
 
+### Enable cross-zone load balancing for the TiDB service LoadBalancer
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:191]: @DanielZhangQD PTAL
```suggestion
### Enable cross-zone load balancing for the `LoadBalancer` TiDB service
```

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
...
+
+Due to an [issue](https://github.com/kubernetes/kubernetes/issues/82595) of AWS Network Load Balancer (NLB), the NLB created for the TiDB service cannot enable cross-zone load balancing. You can manually enable it by taking the following steps:
+
+1. Get the name of the TiDB service NLB:
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:195]:
```suggestion
1. Get the name of the NLB created for the TiDB service:
```

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
...
+
+    This is an example of the output:
+
+    ```
+    kubectl --kubeconfig credentials/kubeconfig_test get svc test-tidb -n test
+    NAME        TYPE           CLUSTER-IP      EXTERNAL-IP                                                                     PORT(S)                          AGE
+    tidb-tidb   LoadBalancer   172.20.39.180   a7aa544c49f914930b3b0532022e7d3c-83c0c97d8b659075.elb.us-west-2.amazonaws.com   4000:32387/TCP,10080:31486/TCP   3m46s
+    ```
+
+    In the `EXTERNAL-IP` field value, the first field that is separated by `-` is the name of NLB. In the above example, it is `a7aa544c49f914930b3b0532022e7d3c`.
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:211]:
```suggestion
    In the `EXTERNAL-IP` field value, the first field that is separated by `-` is the name of NLB. In the example above, the NLB name is `a7aa544c49f914930b3b0532022e7d3c`.
```

File: en/deploy-on-aws-eks.md [hunk 4]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
...
+    This is an example of the output:
+
+    ```
+    aws elbv2 describe-load-balancers | grep a7aa544c49f914930b3b0532022e7d3c
+              "LoadBalancerArn": "arn:aws:elasticloadbalancing:us-west-2:687123456789:loadbalancer/net/a7aa544c49f914930b3b0532022e7d3c/83c0c97d8b659075",
+              "DNSName": "a7aa544c49f914930b3b0532022e7d3c-83c0c97d8b659075.elb.us-west-2.amazonaws.com",
+              "LoadBalancerName": "a7aa544c49f914930b3b0532022e7d3c",
+    ```
+
+    The value of the `LoadBalancerArn` is the NLB LoadBalancerArn.
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:232]:
```suggestion
    The value of the `LoadBalancerArn` field is the NLB LoadBalancerArn.
```

File: en/deploy-on-aws-eks.md [hunk 5]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
...
+
+    ```
+    kubectl --kubeconfig credentials/kubeconfig_test get svc test-tidb -n test
+    NAME        TYPE           CLUSTER-IP      EXTERNAL-IP                                                                     PORT(S)                          AGE
+    tidb-tidb   LoadBalancer   172.20.39.180   a7aa544c49f914930b3b0532022e7d3c-83c0c97d8b659075.elb.us-west-2.amazonaws.com   4000:32387/TCP,10080:31486/TCP   3m46s
+    ```
+
+    In the `EXTERNAL-IP` field value, the first field that is separated by `-` is the name of NLB. In the above example, it is `a7aa544c49f914930b3b0532022e7d3c`.
+
+2. Get the NLB LoadBalancerArn:
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:213]:
```suggestion
2. Get the LoadBalancerArn for the NLB:
```

File: en/deploy-on-aws-eks.md (hunk 1 above)
Comment (DanielZhangQD) [reply en/deploy-on-aws-eks.md:191]:
```suggestion
### Enable cross-zone load balancing for the LoadBalancer of the TiDB service
```

File: en/deploy-on-aws-eks.md [hunk 6]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
...
+
+    This is an example of the output:
+
+    ```
+    kubectl --kubeconfig credentials/kubeconfig_test get svc test-tidb -n test
+    NAME        TYPE           CLUSTER-IP      EXTERNAL-IP                                                                     PORT(S)                          AGE
+    tidb-tidb   LoadBalancer   172.20.39.180   a7aa544c49f914930b3b0532022e7d3c-83c0c97d8b659075.elb.us-west-2.amazonaws.com   4000:32387/TCP,10080:31486/TCP   3m46s
+    ```
+
+    In the `EXTERNAL-IP` field value, the first field that is separated by `-` is the name of NLB. In the example above, the NLB name is `a7aa544c49f914930b3b0532022e7d3c`.
```
Comment (DanielZhangQD) [inline en/deploy-on-aws-eks.md:211]:
```suggestion
    In the value of the `EXTERNAL-IP` field, the first field that is separated by `-` is the name of NLB. In the example above, the NLB name is `a7aa544c49f914930b3b0532022e7d3c`.
```

**Comment by ran-huang** [comment]
@DanielZhangQD @TomShawn PTAL again. Thanks!

File: en/deploy-on-aws-eks.md [hunk 7]
```diff
@@ -188,6 +188,173 @@ You can use the `terraform output` command to get the output again.
     kubectl --kubeconfig credentials/kubeconfig_${eks_name} create -f db-monitor.yaml -n ${namespace}
     ```
 
+### Enable cross-zone load balancing for the LoadBalancer of the TiDB service
+
+Due to an [issue](https://github.com/kubernetes/kubernetes/issues/82595) of AWS Network Load Balancer (NLB), the NLB created for the TiDB service cannot enable cross-zone load balancing. You can manually enable it by taking the following steps:
```
Comment (TomShawn) [inline en/deploy-on-aws-eks.md:193]:
```suggestion
Due to an [issue](https://github.com/kubernetes/kubernetes/issues/82595) of AWS Network Load Balancer (NLB), the NLB created for the TiDB service cannot automatically enable cross-zone load balancing. You can manually enable it by taking the following steps:
```

**Pull request merged**